pkg/fleetautoscalers/controller.go

```diff
@@ -48,7 +50,7 @@ import (
 
 // Controller is a the FleetAutoscaler controller
 type Controller struct {
-	logger                *logrus.Entry
+	baseLogger            *logrus.Entry
```

Comment:
Curious the motivation for the rename to `baseLogger`here - rather than leave it at `logger` ?

I kept looking for an inheritance / composition pattern when i saw "base" :smile: too much OO languages, I expect.

Reply:
`c.logger` is too easy to find and misuse :)